src/Social/Models/Tags.php

```diff
@@ -69,4 +75,17 @@ public function getSource()
     {
         return 'tags';
     }
+
+    /**
+     * Get the interaction key.
+     *
+     * @return string
+     */
+    protected function getInteractionStorageKey(): string
```

Comment:
@arfenis since this metho repeties itself we can also have a trait using the other trait overwritting the method and thus allowing use to use it